src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int bound = 100;
```

Comment:
```suggestion
    private static final int BOUND = 100;
```
Look at how [constants](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names) should be named